site-src/geps/gep-1762.md

```diff
@@ -0,0 +1,312 @@
+# GEP-1757: In Cluster Gateway Deployments
+
+* Status: Provisional
+
+## Overview
+
+Gateway API provides a common abstraction over different implementations, whether they are implemented by cloud load balancers, in-cluster deployments, or other mechanisms. However, many in-cluster implementations have solved some of the same problems in different ways.
+
+Related discussions:
+* [Support cluster-local Gateways](https://github.com/kubernetes-sigs/gateway-api/discussions/1247)
+* [Scaling Gateway Resources](https://github.com/kubernetes-sigs/gateway-api/discussions/1355)
+* [Manual deployments](https://github.com/kubernetes-sigs/gateway-api/issues/1687)
+
```

Comment:
It would appear since this GEP is getting into the `automated` vs. `manual` provisioning of `Gateway` discussions that we should reference  #1687 which was asking for definitions and documentation in this regard (and this GEP would appear to be the first step in supporting that):

```suggestion
* [Static/Unmanaged Gateway Mode](https://github.com/kubernetes-sigs/gateway-api/issues/1687)
```